### PR TITLE
New function to create a handler for already existing users in the cluster.

### PR DESCRIPTION
**Pull request by TejasC88**
Also a new function to create users from a user provided list.

### CODE REVIEW

File: rgw/v2/lib/resource_op.py [hunk 1]
```diff
@@ -74,6 +87,29 @@ def create_tenant_users(no_of_users_to_create, tenant_name, cluster_name='ceph')
     return all_users_details
 
 
+def list_all_users(cluster_name='ceph'):
```
Comment (rakeshgm) [inline rgw/v2/lib/resource_op.py]: @TejasC88 can move this to admin.py

Comment (TejasC88) [reply rgw/v2/lib/resource_op.py]: @rakeshgm Done.

**Comment by rakeshgm** [comment]
@TejasC88 need python3 compatiblity code now.

**Comment by rakeshgm** [comment]
closing this as its become stale.

**Pull request closed without merging**